### DISCUSSION TITLE
computational resources (RAM e CPU)

### DISCUSSION THREAD

**Discussion by miantniocar** [containers/podman#22617] [General]
Hey guys,
I need to collect the computational resources used by podman. But I can't find the process it uses. I'm using this command: "pidof -s podman", but without success. Can you help me please!

**Comment by miantniocar** [comment]
I´m using Debian  12.0 and podman version 4.3.1

**Comment by rhatdan** [comment]
Podman is not a daemon, if you run a container with `-d` podman will exit and the only process monitoring the container is conmon.

If you run the container in foreground mode then Podman will continue to take up resources.
